### ISSUE TITLE
Documentation example fails with AttributeError

### ISSUE THREAD

**Issue by mjstarke** [blaylockbk/goes2go#31]
When installing goes2go [per the documentation](https://blaylockbk.github.io/goes2go/_build/html/index.html#install) using the [provided environment.yml](https://github.com/blaylockbk/goes2go/blob/main/environment.yml) to create a clean conda environment, the [ABI example](https://blaylockbk.github.io/goes2go/_build/html/reference_guide/index.html#rgb-recipes) below does not work:

```
from goes2go.data import goes_latest
import matplotlib.pyplot as plt
import cartopy.crs as ccrs

# Download a GOES ABI dataset
G = goes_latest(product='ABI')

# Make figure on Cartopy axes
ax = plt.subplot(projection=G.rgb.crs )
ax.imshow(G.rgb.TrueColor(), **G.rgb.imshow_kwargs)
ax.coastlines()
```

This fails with

```
Traceback (most recent call last):
  File "~/miniconda3/envs/goes2go/lib/python3.9/site-packages/matplotlib/backends/backend_qt.py", line 456, in _draw_idle
    self.draw()
  File "~/miniconda3/envs/goes2go/lib/python3.9/site-packages/matplotlib/backends/backend_agg.py", line 408, in draw
    self.figure.draw(self.renderer)
  File "~/miniconda3/envs/goes2go/lib/python3.9/site-packages/matplotlib/artist.py", line 74, in draw_wrapper
    result = draw(artist, renderer, *args, **kwargs)
  File "~/miniconda3/envs/goes2go/lib/python3.9/site-packages/matplotlib/artist.py", line 51, in draw_wrapper
    return draw(artist, renderer)
  File "~/miniconda3/envs/goes2go/lib/python3.9/site-packages/matplotlib/figure.py", line 3074, in draw
    mimage._draw_list_compositing_images(
  File "~/miniconda3/envs/goes2go/lib/python3.9/site-packages/matplotlib/image.py", line 131, in _draw_list_compositing_images
    a.draw(renderer)
  File "~/miniconda3/envs/goes2go/lib/python3.9/site-packages/matplotlib/artist.py", line 51, in draw_wrapper
    return draw(artist, renderer)
  File "~/miniconda3/envs/goes2go/lib/python3.9/site-packages/cartopy/mpl/geoaxes.py", line 558, in draw
    return matplotlib.axes.Axes.draw(self, renderer=renderer, **kwargs)
  File "~/miniconda3/envs/goes2go/lib/python3.9/site-packages/matplotlib/artist.py", line 51, in draw_wrapper
    return draw(artist, renderer)
  File "~/miniconda3/envs/goes2go/lib/python3.9/site-packages/matplotlib/axes/_base.py", line 3107, in draw
    mimage._draw_list_compositing_images(
  File "~/miniconda3/envs/goes2go/lib/python3.9/site-packages/matplotlib/image.py", line 131, in _draw_list_compositing_images
    a.draw(renderer)
  File "~/miniconda3/envs/goes2go/lib/python3.9/site-packages/matplotlib/artist.py", line 51, in draw_wrapper
    return draw(artist, renderer)
  File "~/miniconda3/envs/goes2go/lib/python3.9/site-packages/cartopy/mpl/feature_artist.py", line 147, in draw
    extent = ax.get_extent(feature_crs)
  File "~/miniconda3/envs/goes2go/lib/python3.9/site-packages/cartopy/mpl/geoaxes.py", line 814, in get_extent
    p = self._get_extent_geom(crs)
  File "~/miniconda3/envs/goes2go/lib/python3.9/site-packages/cartopy/mpl/geoaxes.py", line 821, in _get_extent_geom
    with self.hold_limits():
  File "~/miniconda3/envs/goes2go/lib/python3.9/contextlib.py", line 119, in __enter__
    return next(self.gen)
  File "~/miniconda3/envs/goes2go/lib/python3.9/site-packages/cartopy/mpl/geoaxes.py", line 491, in hold_limits
    self._autoscaleXon, self._autoscaleYon)
AttributeError: 'GeoAxesSubplot' object has no attribute '_autoscaleXon'
```

**Comment by blaylockbk** [comment]
I'm not sure why this isn't working for you. I got a result with that exact snippet.

![image](https://user-images.githubusercontent.com/6249613/197054918-5fa9517b-bd27-43b0-824c-83311eebbc68.png)

I suspect there are some issues with cartopy. I have cartopy v0.18.0 currently installed.

FYI: I did release a new version to PyPI today (v2022.10.0), but I only changed the installation setup.cfg file. It might be worth a try.
